app/assets/stylesheets/events.scss

```diff
@@ -314,7 +314,7 @@ label {
   border-radius: 0.5rem;
 }
 
-input[type="datetime-local"].dropdown-select, select#call_position_id {
+input.dropdown-select, select#call_position_id, select#venue_state {
```

Comment:
Avoid using id selectors<br>Avoid qualifying class selectors with an element.<br>Avoid qualifying id selectors with an element.<br>Selector `call_position_id` should be written in lowercase with hyphens<br>Selector `venue_state` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own line